### PR TITLE
Fix/snackbar

### PR DESCRIPTION
**Pull request by edentorres**
## Description
Describe why we need these changes in the lib.
Try to answer these questions:
- What is it about?
 Fixes snackbarsQueue logic when a snackbar is no longer visible
- How does it work?
## Affected component
Is highly probable that this new code affects directly one (or more?) components inside this lib. Tell us who they are!
- AndesSnackbar
## RFC Link
If this change was discussed on RFC please paste link here.
## Screenshots / GIFs
This is a UI library, delight us with some stunning images.
## Checks
Are you sure you followed all those steps? If so, repeat after me "I solemnly swear that ...":
   - [ ] I have coded an example inside the Demo App,
   - [ ] I have added proper tests,
   - [X] I have modified the Changelog.md file,
   - [ ] I have updated GitHub wiki,
   - [ ] I have the explicit approval of one or more members of the UX Team,
   - [ ] I have checked that this code is ObjC compliant.
   - [ ] I have checked that all the new public enums conform to AndesEnumStringConvertible.
## Change type
This is easy, let us know what this code is about:
   - [ ] This code adds a new component
   - [X] This code includes improvements to an existent component
   - [ ] This code improves or modifies ONLY the Demo App.

### CODE REVIEW

**Comment by furyGorgoryAdmin** [comment]
### Informative Menu <h2>  ### 

### Use any of these tags in a pull request description or leave a comment ### 
| Tag | Description |
|---|---|
| **[canary]**| Creates a build that can only be deployed with canary strategy. | 
| **[build]** |  Creates version or build from a branch that has been tagged as "no releasable". | 
| **[name:version-name]** |Specify the name of the build. For example: [build] [name:1.2.3-name] or [canary] [name:1.2.3-name].|
| **[major]** | To automatically increase the first position of the version. For example: [build] [major] or [canary] [major].|
| **[minor]** | To automatically increase the second position of the version. For example: [build] [minor] or [canary] [minor]. |
| **[patch]** | To automatically increase the third position of the version. For example: [build] [patch] or [canary] [patch].|  

[For more information about ci-cd](http://docs.furycloud.io/?lang=esp&section=businessassurance)
[For more information about semver](https://semver.org/#summary)

File: LibraryComponents/Classes/AndesSnackbar/Manager/AndesSnackbarManager.swift [hunk 1]
```diff
@@ -106,6 +110,8 @@ class AndesSnackbarManager {
 
     func dismissSnackbarAnimated(snackbar: AndesSnackbar) {
         guard let superview = snackbar.superview else {
+            self.snackbarsQueue.removeFirst()
+            self.showNextSnackbar()
```
Comment (samuelsainz) [inline LibraryComponents/Classes/AndesSnackbar/Manager/AndesSnackbarManager.swift:114]: @edentorres Faltaría invalidar el timer en caso de que aún esté vivo, así evitamos que se vuelva a ejecutar el dismiss si este dismiss se hizo manualmente

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## Unpublished
+### 🛠 Bug fixes
+
+- AndesSnackbar: Fixes snackbarsQueue logic when a snackbar is no longer visible| Authors: [@EdenTorres](https://github.com/mercadolibre/fury_andesui-ios)
```
Comment (samuelsainz) [inline CHANGELOG.md:4]: Te animas a poner este cambio en la `3.6.0`? Mala mia que la dejé sin la etiqueta de `Unpublished` 😅

File: LibraryComponents/Classes/AndesSnackbar/Manager/AndesSnackbarManager.swift (hunk 1 above)
Comment (edentorres) [reply LibraryComponents/Classes/AndesSnackbar/Manager/AndesSnackbarManager.swift:115]: dale!

**Pull request merged**